src/desktop/index.ts

```diff
@@ -4,6 +4,10 @@ import * as globalReactModules from "desktop/components/react/stitch_components"
 
 const app = express()
 
+// FIXME: Move this
+import { app as shortcuts } from "v2/System/Server/shortcuts"
+app.use(shortcuts)
```

Comment:
So this shouldn't be here but bizarrely I was unable to actually mount it anywhere else and it still work. This is one of those things that will require a rubber duck pair because if I try to explain it just doesn't make sense.